core/src/main/java/com/amazon/opendistroforelasticsearch/sql/storage/StorageEngine.java

```diff
@@ -23,5 +23,5 @@
   /**
    * Get {@link Table} from storage engine.
    */
-  Table getTable(String name);
+  Table getTable(String name, Integer size);
```

Comment:
The orignal definition of getTable is get Table from storage engine by name, the size paramater doesn't make sense in here.